packages/libp2p/src/transport-manager.ts

```diff
@@ -262,12 +262,22 @@ export class DefaultTransportManager implements TransportManager, Startable {
    * If a transport has any running listeners, they will be closed.
    */
   async remove (key: string): Promise<void> {
-    log('removing %s', key)
+    const listeners = this.listeners.get(key) ?? []
+    log('removing transport %s', key)
```

Comment:
I don't think we need these logs enabled by default.
```suggestion
    log.trace('removing transport %s', key)
```